.rubocop.yml

```diff
@@ -34,359 +33,24 @@ plugins:
-  Enabled: false
-Style/RedundantFetchBlock: # (new in 0.86)
-  Enabled: false
-Style/RedundantFileExtensionInRequire: # (new in 0.88)
-  Enabled: true
-Style/RedundantRegexpCharacterClass: # (new in 0.85)
-  Enabled: true
-Style/RedundantRegexpEscape: # (new in 0.85)
-  Enabled: false
-Style/RedundantSelfAssignment: # (new in 0.90)
-  Enabled: true
-Style/SingleArgumentDig: # (new in 0.89)
-  Enabled: true
-Style/SlicingWithRange: # (new in 0.83)
-  Enabled: false
-Style/SoleNestedConditional: # (new in 0.89)
-  Enabled: false
-Style/StringConcatenation: # (new in 0.89)
-  Enabled: false
-
-Rails/ContentTag:
-  Enabled: false
-Rails/UniqueValidationWithoutIndex:
-  Enabled: false
-Performance/Detect:
-  Enabled: false
-Rails/RedundantForeignKey:
-  Enabled: false
-
-Rails/ActiveRecordCallbacksOrder: # (new in 2.7)
-  Enabled: false
-Rails/AfterCommitOverride: # (new in 2.8)
-  Enabled: true
-Rails/FindById: # (new in 2.7)
-  Enabled: true
-Rails/Inquiry: # (new in 2.7)
-  Enabled: true
-Rails/MailerName: # (new in 2.7)
-  Enabled: true
-Rails/MatchRoute: # (new in 2.7)
-  Enabled: true
-Rails/NegateInclude: # (new in 2.7)
-  Enabled: false
-Rails/Pluck: # (new in 2.7)
-  Enabled: false
-Rails/PluckInWhere: # (new in 2.7)
-  Enabled: true
-Rails/RenderInline: # (new in 2.7)
-  Enabled: true
-Rails/RenderPlainText: # (new in 2.7)
-  Enabled: true
-Rails/ShortI18n: # (new in 2.7)
-  Enabled: true
-Rails/SquishedSQLHeredocs: # (new in 2.8)
-  Enabled: true
-Rails/WhereExists: # (new in 2.7)
-  Enabled: true
-Rails/WhereNot: # (new in 2.8)
-  Enabled: true
-Performance/AncestorsInclude: # (new in 1.7)
-  Enabled: true
-Performance/BigDecimalWithNumericArgument: # (new in 1.7)
-  Enabled: true
-Performance/RedundantSortBlock: # (new in 1.7)
-  Enabled: true
-Performance/RedundantStringChars: # (new in 1.7)
-  Enabled: true
-Performance/ReverseFirst: # (new in 1.7)
-  Enabled: true
-Performance/SortReverse: # (new in 1.7)
-  Enabled: true
-Performance/Squeeze: # (new in 1.7)
-  Enabled: true
-Performance/StringInclude: # (new in 1.7)
-  Enabled: false
-Performance/Sum: # (new in 1.8)
-  Enabled: false
-
-Naming/VariableNumber:
-  Enabled: false
-Performance/CollectionLiteralInLoop:
-  Enabled: true
-Rails/Date:
-  Enabled: true
-Lint/RedundantCopDisableDirective:
-  Enabled: false
-Lint/RedundantCopEnableDirective:
-  Enabled: false
-Rails/RelativeDateConstant:
-  Enabled: false
-Style/ZeroLengthPredicate:
-  Enabled: false
-Style/NumericLiterals:
-  Enabled: false
-Performance/TimesMap:
-  Enabled: false
-
-### RSpec cops we're configuring for our project ###
-
-RSpec/ChangeByZero:
-  Enabled: false
-RSpec/DescribedClass:
-  Enabled: false
-RSpec/LeadingSubject:
-  Enabled: false
-RSpec/ScatteredLet:
-  Enabled: false
-RSpec/NestedGroups:
-  Enabled: false
-RSpec/VariableName:
-  AllowedPatterns:
-    - _£_
-RSpec/ScatteredSetup:
-  Enabled: false
-RSpec/ItBehavesLike:
-  EnforcedStyle: 'it_behaves_like' # rather than 'it_should_behave_like'
```

Comment:
```
RSpec/ItBehavesLike:
  EnforcedStyle: 'it_behaves_like' # rather than 'it_should_behave_like'
RSpec/MultipleExpectations:
  Max: 12 # max expectations per test
RSpec/LetSetup:
  Enabled: false
```

I remember adding these.
RSpec/ItBehavesLike - to make sure we don't use it_should_behave_like, just it_behaves_like
RSpec/MultipleExpectations - good for when checking for multiple form fields on a page - can also be quicker to have multiple expectations in one test (less setup/tare down).
RSpec/LetSetup - I use this pattern, be it right or wrong, I feel like it reads / writes better on the whole. Can be tricky to debug sometimes. The alternative can lead to quite a bit of code duplication.